### PR TITLE
BBaker: Shadui Redesign of roundActionCard Component  

### PR DESCRIPTION
**Pull request by BBtxt**
Clean and Simple design of the RoundActionCard.

1. Remove divs and styling. Will style once skeleton is completed. 
2. Pulled song information from the roundProvider() on app/page.tsx . Created a new type interface for the song object to be used in the RoundActionCard component.
3. Refactored the UseBlurb to return phaseStatus and phaseBlurb.
4. Added little logic to change placement of the phaseBlurb depending on the phase.
5. Changed CTA button Elements to ShadUI Components .
6. Renamed "Sign up" button to "Login" very confusing since sign up is on of the phases and this is referring to if the user is auth'd.

Question: 
Do we need labelContent() function it seems like it provided redundant information? 

I'm hoping we'll get less calls to the dB now that we are calling the DB twice on the home page load due to having two roundProvider calls in app/page.tsx and heroAction.tsx. Does this make sense? lol

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -21,18 +21,18 @@ const Homepage = async () => {
       </Head>
       <div className="flex justify-center items-center h-4/5">
       <Hero />
-
       </div>
      
-      {/* <div className="mt-8 md:-mt-20 mb-12">
```
Comment (nspilman) [inline app/page.tsx:27]: nit - remove?

File: app/voting/Homepage/RoundActionCard/RoundActionCard.tsx [hunk 2]
```diff
@@ -60,50 +71,59 @@ export const RoundActionCard = ({
...
-        <>{`Next round starts in ${phaseEndsDaysFromToday} days`}</>
-      ) : (
-        <>Notify me when next round starts</>
-      );
-    }
-  })();
-
-  const blurb = useBlurb({ phase, roundId, phaseEndsDatelabel });
+
+  // Do we need this????
```
Comment (nspilman) [inline app/voting/Homepage/RoundActionCard/RoundActionCard.tsx:75]: We should discuss it, but you can remove it for now. This is helper text, but I like the idea of things being so clear it's unnecessary. We can add something similar back later.

**Review by nspilman — APPROVED** [review]
LGTM! Mind posting some screenshots so I can see how this looks? I know we're pre-styling, but I'm curious.

File: app/page.tsx (hunk 1 above)
Comment (BBtxt) [reply app/page.tsx:27]: Want me to remove the whole div?

File: app/voting/Homepage/RoundActionCard/RoundActionCard.tsx (hunk 2 above)
Comment (BBtxt) [reply app/voting/Homepage/RoundActionCard/RoundActionCard.tsx:75]: Done and dusted

**Comment by BBtxt** [comment]
Screenshot 2024-05-03 at 10 04 17
Screenshot 2024-05-03 at 10 03 45

**Pull request merged**